Data Structure & Algorithm/reverseArray.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/*
+ * Complete the 'reverseArray' function below.
+ *
+ * The function is expected to return an INTEGER_ARRAY.
+ * The function accepts INTEGER_ARRAY a as parameter.
+ */
+
+function reverseArray($a) {
+    if (empty($a)) {
+        return $a;
+    } else {
```

Comment:
FYI: this else block does no matter because you have a `return statement` in if, it means you will have the same behavior with code:
```
function wow() {
  if (condition) { return; }
  doSomeOtherOperations...
  return;

}
```